GiniVision/Classes/Networking/AccountingDocumentService.swift

```diff
@@ -6,19 +6,21 @@
 //
 
 import Foundation
-import Gini_iOS_SDK
+import Gini_iOS_SDK
```

Comment:
Why is this import needed now?

Reply:
It was already there. Not sure why the repo changed this file at that lien, myabe a line break or similar.